source/guide_codimd.rst

```diff
@@ -0,0 +1,309 @@
+		CMD_ALLOW_EMAIL_REGISTER=false,
+		CMD_IMAGE_UPLOAD_TYPE=filesystem,
+		CMD_PROTOCOL_USESSL=true,
+		CMD_DB_URL="mysql://<username>:<mysql_pw>@localhost:3306/<username>_codimd",
+
+	directory=/home/<username>/codimd
+	command=node app.js
+
+In our example this would be:
+
+.. code-block:: ini
+
+	[supervisord]
+	loglevel=warn
+
+	[program:codimd]
+	environment=
+		CMD_SESSION_SECRET="extremerandom",
+		CMD_DOMAIN="isabell.uber.space",
+		CMD_PORT=9000,
+		NODE_ENV="production",
+		CMD_USECDN=false,
+		CMD_ALLOW_ANONYMOUS=false,
+		CMD_ALLOW_ANONYMOUS_EDITS=true,
+		CMD_ALLOW_FREEURL=true,
+		CMD_DEFAULT_PERMISSION=private,
+		CMD_SESSION_LIFE=1209600000,
+		CMD_EMAIL=true,
+		CMD_ALLOW_EMAIL_REGISTER=false,
+		CMD_IMAGE_UPLOAD_TYPE=filesystem,
+		CMD_PROTOCOL_USESSL=true,
+		CMD_DB_URL="mysql://isabell:MySuperSecretPassword@localhost:3306/isabell_codimd",
+
+	directory=/home/isabell/codimd
+	command=node app.js
+
+
+
+Replace the values in ``CMD_SESSION_SECRET``, ``CMD_DOMAIN``, ``CMD_PORT`` and ``CMD_DB_URL`` and you're good to go!
+
+See `here <https://github.com/hackmdio/codimd#environment-variables-will-overwrite-other-server-configs>`_ for a detailed look at what these options do.
+
+
+Once you run the following step, this will create a CodiMD instance without reliance on third-party CDN servers, which is private to people who had an account created via CLI.
+
+
+
+Finishing installation
+======================
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ supervisorctl reread
+  [isabell@stardust ~]$ supervisorctl update
+  [isabell@stardust ~]$ 
+
+Add a user
+----------
+
+Since we don’t want to run a public instance and have disabled registration, we use the CLI to add a user:
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ cd ~/codemd
+  [isabell@stardust ~]$ CMD_DB_URL="mysql://<username>:<mysql_pw>@localhost:3306/<username>_codimd" bin/manage_users --add isabell@uberspace.de
+  [isabell@stardust ~]$ 
+
+It will prompt you for a password now.
+
+.. warning:: 
+
+  Don’t forget your password, neither the site nor the CLI currently seem to offer a way to reset it!
+
+
+Configure web server
+--------------------
+
+In order for your CodeMD instance to be reachable from the web, you need to put a file called ``.htaccess`` into your ``~/html`` folder (or any other DocumentRoot, see the `document root`_ for details), with the following content:
+
+.. code-block:: ini
+
+  DirectoryIndex disabled
+
+  RewriteEngine On
+  RewriteCond %{HTTPS} !=on
+  RewriteCond %{ENV:HTTPS} !=on
+  RewriteRule .* https://%{SERVER_NAME}%{REQUEST_URI} [R=301,L]
+  RewriteRule (.*) http://localhost:<PORT>/$1 [P]
+
+Again, don't forget to fill in your port number in the last line!
+
+And you're done!
+
+Updates
+=======
+
+.. warning:: 
+  I have not actually tested this so far!
+
+If you want to update your CodiMD installation, follow `these instruction <https://github.com/hackmdio/codimd#upgrade>`_, and merge it with this manual. Good luck.
```

Comment:
Imagine trying to follow this guide, something does not work and then there's a "good luck" somewhere in the text. I would be pissed tbh.